src/nonStrict.ts

```diff
@@ -0,0 +1,12 @@
+import { internalRecord } from './record'
+import { Runtype, RuntypeUsageError } from './runtype'
+
+export function nonStrict<T>(original: Runtype<T>): Runtype<T> {
+  const fields = (original as any).fields
+
+  if (!fields) {
+    throw new RuntypeUsageError('expected a record runtype')
+  }
+
+  return internalRecord(fields, true)
+}
```

Comment:
Like pick, omit, etc., nonStrict takes in a runtype, checks that its a record runtype (i.e. has a `fields` property), and then uses `internalRecord` to create a new runtype, this time passing in `true` as `internalRecord`'s `isNonStrict` argument.

Reply:
should `nonStrict` be in `record.ts` instead, given its connection with that runtype? or like omit, pick, etc. should it be in its own file (like it is now)?

Reply:
- [ ] Could you though please add a short /** ... */ function definition comment to nonStrict.

Reply:
@hoeck Added a JSDoc comment 👍 I've tried to keep it as consistent as possible with other comments, while also trying to make the description clear and concise. Let me know what you think.

I primarily use VS Code, and this is what the JSDoc comment looks like there:

<details>
<summary>Screenshot</summary>

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/8270635/221395560-4ead0610-8d94-4f6f-9a33-7ed985bbfcc7.png">

</details>

Reply:
Thanks, that docstring looks good and describes the nonStrict modifier very well.